pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
```

Comment:
Seems to have automatically moved from bottom to top - might different pnpm version